test-app-STM32F4-measured-boot/README.md

```diff
@@ -74,11 +74,11 @@ Note: Make sure the Ground connection between your USB-UART converter is connect
 
 Before compiling make sure the git submodules are initialized and updated correctly. Use the following commands to make sure:
 
-`wolfboot-examples/$ git submodule --init --update`
+`wolfboot-examples/$ git submodule update --init `
```

Comment:
Thank you for the report. This should be `git submodule update --init --recursive`. I'll treat this as a bug report and we will fix ourselves.